finalPrep/modZeroHero.js

```diff
@@ -2,45 +2,97 @@
 
 // Declare two variables - heroName AND specialAbility - set to strings
 
+var heroName = 'Red Rush';
+
+var specialAbility = 'Super Speed';
+
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = "There is nothing to fear, " + heroName + " is here!";
+
+var catchphrase = `My ${specialAbility} is all we NEED!`;
+
 // Declare two variables - power AND energy - set to integers
 
+var power = 50;
+
+var energy = 1000000;
+
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
+
+var fullPower = power * 500;
+
 //   fullEnergy should add 150 to your current energy
 
+var fullEnergy = energy + 150;
+
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
 
+var isHuman = true;
+
+var identityConcealed = false;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
+
+var archEnemies = ['Slow Blue', 'Cool Rush', 'Micheal Keaton'];
+
 //   sidekicks should be an array of at least 3 different sidekick strings
 
+var sidekicks = ['Green Rush', 'Backpack Kid', 'Rush Jr.'];
+
 // Print the first sidekick to your console
 
+console.log(sidekicks[0]);
+
 // Print the last archEnemy to the console
 
+console.log(sidekicks[2]);
```

Comment:
Nice job with your bracket notation. A next level is to think about how to print the last element regardless of how long that array is.